src/users/__tests__/userSelectors-test.js

```diff
@@ -208,3 +209,26 @@ describe('getUsersById', () => {
     expect(result).toEqual(expectedResult);
   });
 });
+
+describe('getUsersSansMe', () => {
+  test('returns all users but current user', () => {
```

Comment:
I guess
`return all users except self` or `return all users except` will be better name?

Reply:
Done :+1: .